### PR TITLE
New Package:  jwm-s1103

### PR DESCRIPTION
**Pull request by silvernode**

### CODE REVIEW

File: srcpkgs/jwm/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'jwm'
+
+pkgname="jwm"
+version="s1103"
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="automake cairo-devel libjpeg-turbo-devel
```
Comment (xtraeme) [inline srcpkgs/jwm/template]: automake must be in hostmakedepends instead.

File: srcpkgs/jwm/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+pkgname="jwm"
+version="s1103"
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="automake cairo-devel libjpeg-turbo-devel 
+libpng-devel libXext-devel libXrender-devel 
+libXmu-devel libXft-devel libXinerama-devel
+libXpm-devel fribidi-devel"
+short_desc="JWM is a light-weight window manager for the X11 Window System"
```
Comment (xtraeme) [inline srcpkgs/jwm/template]: please remove "JWM is"... it's redundant.

**Comment by silvernode** [comment]
it's been awhile, I pretty sure I just forgot 32 bit deps

**Comment by xtraeme** [comment]
You are also missing `conf_files="/etc/system.jwmrc"`

**Comment by xtraeme** [comment]
Also, you should replace `cairo-devel` with `librsvg-devel`... otherwise `cairo` support is disabled.

**Pull request merged**